src/test/scala/beam/agentsim/infrastructure/ZonalParkingManagerSpec.scala

```diff
@@ -451,10 +451,11 @@ class ZonalParkingManagerSpec
         zonesMap,
         SpaceTime(new Coord(170308.0, 2964.0), 0),
         "4",
-        ParkingZone.createId("cs_default(Any)_4_Residential_NA_Block_199_1144"),
+        ParkingZone.createId("cs_default(Any)_4_Public_NA_Block_199_1144"),
```

Comment:
The parking we get in response is now public because of the change in `beam.agentsim.infrastructure.ParkingFunctions#getPreferredParkingTypes` method.